taskfiles/build.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+tasks:
+  genpartial:
+    dir: ./genpartial
+    sources:
+    - ./genpartial/**.go
+    generates:
+    - ../.build/genpartial
+    cmds:
+    - go build -o ../.build/genpartial .
+
+  genschema:
+    dir: genschema
+    cmds:
+    - go build -o ../.build/genschema .
+
+  genpartial:
+    dir: genpartial
+    cmds:
+    - go build -o ../.build/genpartial .
+
+  gotohelm:
+    dir: gotohelm
+    sources:
+    - ./genpartial/**.go
+    cmds:
+    - go build -o ../.build/gotohelm .
+
+  licenseupdater:
+    dir: licenseupdater
+    cmds:
+    - go build -o ../.build/licenseupdater .
```

Comment:
If the `update-licenses` target has the following command
```
go run -C licenseupdater ./... -config ../.licenseupdater.yaml
```
then I don't understand why `licenseupdater` needs to be build.